src/mozanalysis/stats.py

```diff
@@ -1,66 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Is it premature to remove stats.py?

Reply:
I want to make several breaking changes to `stats.py` so I don't want people to start using this version.

After this PR, the next step is to issue another PR that adds back some stats functionality (including some lines of code lifted from this original file). This second PR will also lead to a lot of discussion, but the issues there are mostly orthogonal to the issues in this PR so I'd rather have these discussions sequentially rather than simultaneously.